services/web/server/tests/integration/01/test_garbage_collection.py

```diff
@@ -161,7 +161,17 @@ def client(
     )
 
 
-################ utils
+@pytest.fixture
+def disable_garbage_collector_background_task(mocker):
```

Comment:
not sure I understand why this is required.

Reply:
It is documented in the function, I quote: patch the setup of the garbage collector **so we can call it manually**